.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: Continuous Integration
+on: [push, pull_request]
+jobs:
+  ruby:
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [2.4, 2.5, 2.6]
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Setup Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+    - name: Install dependencies
```

Comment:
I'm vaguely surprised you don't need to force cmake to be installed, but this looks good.